app/controllers/admin/ticket_scannings_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Admin
+  class TicketScanningsController < Admin::BaseController
+    before_action :authenticate_user!
+    before_action :find_physical_ticket_by_token
```

Comment:
I am not using `load_resource :ticket_scanning, find_by: :token` because it raises an record not found exception if the resource is not found as it is implemented by using `find_by!`.